tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -87,7 +87,16 @@ func (rb *IptablesBuilderImpl) AppendRuleV6(chain string, table string, params .
 
 func (rb *IptablesBuilderImpl) BuildV4() []string {
 	rules := []string{}
```

Comment:
Why not `nil`?
It would actually be better if you could find an upper bound to this slice, and allocate it once here:
```suggestion
	rules := make([]string, 0, ???)
```